console/src/components/ApiExplorer/GraphiQLWrapper.js

```diff
@@ -51,15 +60,28 @@ class GraphiQLWrapper extends Component {
     if (!this.state.error && this.props.numberOfTables !== 0) {
       if (this.state.queries) {
         content = (
-          <GraphiQL fetcher={graphQLFetcher} query={this.state.queries} />
+          <GraphiQL
+            fetcher={graphQLFetcher}
+            analyzeFetcher={analyzeFetcherInstance}
```

Comment:
Add server version check to display Analyze button.

Reply:
Should get resolved in the latest push